### DISCUSSION TITLE
Use reverse proxy on XAMPP

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21426] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>Currently, I’m running a local media server on my home computer. it’s using the address of <a href="http://localhost:8096" rel="noopener nofollow ugc">http://localhost:8096</a>. I’ve Used reverse proxy to access the website via <a href="http://localhost" rel="noopener nofollow ugc">http://localhost</a> address.</p>
<p>The code I’ve used for the vhost.conf file is down below.</p>
<pre><code class="lang-auto">&lt;VirtualHost *:80&gt;
	ProxyRequests On
	&lt;Proxy&gt;
		Order deny,allow
		Allow from all
	&lt;/Proxy&gt;
	ProxyPass / http://localhost:8096/
	ProxyPassReverse / http://localhost:8096/
	ServerName localhost
	Header set Access-Control-Allow-Origin "*"
&lt;/VirtualHost&gt;
</code></pre>
<p>I want to know how can I use my domain name instead of “localhost” to access my server via reverse proxy. Can someone help me, please?</p>
<p>Thank you.</p>

**Comment by airtower-luna** [comment]
<p>I assume you mean you want to reach the reverse proxy under your domain name. In the reverse proxy configuration you only need to adjust the <code>ServerName</code>. And then you need to point the DNS entry for your domain at the reverse proxy.</p>
<p>That said, for a server reachable outside localhost you really should set up TLS for security. :slightly_smiling_face:</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello, Thank you so much for your reply. I happy to tell you it actually works very well. Again thank you so much :slight_smile:</p>
